packages/browserslist-config-terra/README.md

```diff
@@ -0,0 +1,63 @@
+# BrowsersList Config Terra
+
+[![NPM version](https://badgen.net/npm/v/@carbon/browserslist-config-terra)](https://www.npmjs.org/package/@carbon/browserslist-config-terra)
+[![Build Status](https://badgen.net/travis/cerner/terra-toolkit)](https://travis-ci.com/cerner/terra-toolkit)
+
+This configuration reflects Terra's supported browser policy for their UI library and build tools.
+
+## What is Browserslist
+
+[Browserslist](https://github.com/ai/browserslist) is a library to share a browsers list between different front-end tools, like Autoprefixer, Eslint, and Stylelint.
+
+* Valid Browserslist query syntax validation [browserl.ist](http://browserl.ist)
+* ["Browserslist is a Good Idea"](https://css-tricks.com/browserlist-good-idea/) (blog post by [@chriscoyier](https://github.com/chriscoyier))
+
+## Supported Browsers
+
+| Browser                     | Version |
+|-----------------------------|---------|
+| Chrome & Chrome for Android | Current |
+| Edge                        | Current |
+| Firefox                     | Current |
+| Internet Explorer           | 10 & 11 |
+| Safari & Mobile Safari      | Current |
```

Comment:
should we add iOS >= 12 to match https://github.com/cerner/terra-toolkit/pull/402/files#diff-aa4b7227170ea8c9b43723be1f1ca4e8R2?

Reply:
https://github.com/cerner/terra-toolkit/pull/402/commits/0002560b7168f2a7524286d1f774b6bf9908ea50